lib/pacio_test_kit/common_tests/update_test.rb

```diff
@@ -0,0 +1,52 @@
+require_relative '../pacio_profiles'
+require_relative '../interactions_test'
+
+module PacioTestKit
+  class UpdateTest < Inferno::Test
+    include PacioTestKit::PacioProfiles
+    include PacioTestKit::InteractionsTest
+
+    title 'Resources returned in previous read tests can be updated'
+    id :pacio_resource_update
+    description %(
+      This test verifies resources returned from the read tests are able to be updated.
+
+      This test will be skipped if no resource read requests were made in
+      previous tests or if all requests were unsuccessful.
+    )
+
+    input :id_to_update,
+          title: 'The id of the FHIR resource to update on the server',
+          description: 'Provide a string id of a resource to update on the server.'
+
+    input :new_resource,
+          title: 'The resource to update the FHIR resource on the server',
+          description: 'Provide a FHIR resource to update on the server.'
+
+    def resource_type
+      config.options[:resource_type]
+    end
+
+    def tag
+      config.options[:profile]
+    end
+
+    run do
+      load_tagged_requests(tag)
+      skip_if requests.blank?, "No #{tag} resource read request was made in previous tests as expected."
+
+      successful_requests = requests.select { |request| request.verb.downcase == 'read' && request.status == 200 }
```

Comment:
Do not need to check for the `verb` because only requests from the `read_test` are tagged.
> Search requests will also be tagged, but the `verb` for search is still `read`.